src/engine/shared/console.cpp

```diff
@@ -109,6 +109,46 @@ const IConsole::CCommandInfo *CConsole::FirstCommandInfo(int AccessLevel, int Fl
 	return nullptr;
 }
 
+int CConsole::AccessLevelToInt(const char *pAccessLevel)
+{
+	// alias for old legacy integer access levels
+	if(!str_comp_nocase(pAccessLevel, "0"))
+		return ACCESS_LEVEL_ADMIN;
+	if(!str_comp_nocase(pAccessLevel, "1"))
+		return ACCESS_LEVEL_MOD;
+	if(!str_comp_nocase(pAccessLevel, "2"))
+		return ACCESS_LEVEL_HELPER;
+	if(!str_comp_nocase(pAccessLevel, "3"))
+		return ACCESS_LEVEL_USER;
+
+	// new string access levels
+	if(!str_comp_nocase(pAccessLevel, "admin"))
+		return ACCESS_LEVEL_ADMIN;
+	if(!str_comp_nocase(pAccessLevel, "mod") || !str_comp_nocase(pAccessLevel, "moderator"))
+		return ACCESS_LEVEL_MOD;
+	if(!str_comp_nocase(pAccessLevel, "helper"))
+		return ACCESS_LEVEL_HELPER;
+	if(!str_comp_nocase(pAccessLevel, "all"))
+		return ACCESS_LEVEL_USER;
+	return -1;
+}
+
+const char *CConsole::AccessLevelToString(int AccessLevel)
+{
+	switch(AccessLevel)
+	{
+	case ACCESS_LEVEL_ADMIN:
+		return "admin";
+	case ACCESS_LEVEL_MOD:
+		return "moderator";
+	case ACCESS_LEVEL_HELPER:
+		return "helper";
+	case ACCESS_LEVEL_USER:
+		return "all";
+	}
+	return nullptr;
```

Comment:
```suggestion
	dbg_assert(false, "invalid access level");
```
Be strict by default.